### PR TITLE
Gruvbox Dark Theme

### PR DESCRIPTION
**Pull request by ha7ak3**
For all gruvbox lovers. Maybe it needs a few tweaks but it looks great imo.

### CODE REVIEW

**Review by redromnon — APPROVED** [review]
Looks great! Thanks!

**Comment by redromnon** [comment]
Well, the theme doesn't load when I added it to Heroic. There is a small change that needs to be done.

File: Gruvbox Dark/gruvbox_dark.css [hunk 1]
```diff
@@ -0,0 +1,36 @@
+body.gruvboxdark {
```
Comment (redromnon) [inline Gruvbox Dark/gruvbox_dark.css]: Please rename this to `gruvbox_dark`. This should be the same as the filename.

**Comment by ha7ak3** [comment]
Fixed it!

**Review by redromnon — APPROVED** [review]

**Comment by redromnon** [comment]
Great! Thanks!

**Pull request merged**